backend-ruby/lib/photobank/api.rb

```diff
@@ -1,14 +1,18 @@
 # frozen_string_literal: true
 
 require "grape"
+require_relative "v1/base"
 
 module Photobank
-  class Web < Grape::API
-    version "v1", using: :path
+  class API < Grape::API
+    version "v1", using: :path, vendor: 'Photobank'
```

Comment:
I don't think vendor does anything here, it's used for the version header, we're using the path.